packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -1018,6 +1047,16 @@ contract Colosseum_ValidatorSystemUpgrade_Test is Colosseum_Initializer {
         return arr;
     }
 
+    function _firstSegments() private view returns (bytes32[] memory) {
```

Comment:
What does it mean of firstSegments?

Reply:
First segments of the challenge. I added it since the code block is repeated several times.

Reply:
Definitely, it's good to have this function. But I wonder the "first" means. How about `_getSegments` like `_getProof`?

Reply:
Then I'll rename it `_getFirstSegments`!

Reply:
Ah I see. That's much clearer!